styles.css

```diff
@@ -0,0 +1,6 @@
+body { font-family: 'Roboto', sans-serif; }
+
+.section__title,
+.header__logo { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/html_2.0/pullRequest?prid=6290605)